README.md

```diff
@@ -364,11 +364,16 @@ Other Style Guides
     ```
 
   <a name="arrays--from"></a><a name="4.4"></a>
-  - [4.4](#arrays--from) To convert an array-like object to an array, use [Array.from](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from).
+  - [4.4](#arrays--from) To convert an array-like object to an array, use spreads `...` instead of [Array.from](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from).
```

Comment:
this is good, but we should also include a "good" example that uses `Array.from` with a mapper function - specifically, `Array.from(foo, bar)` is much much better than `[...foo].map(bar)`.